src/client/components/componentWithSliders/allSliders/button/button.css

```diff
@@ -1,53 +1,95 @@
 .PlayButton {
-  border-radius: 4px;
-  margin: auto 30px;
+  font-family: var(--font-ordinary);
+  font-size: var(--fontSize-text);
+  color: var(--mainText-ordinary-white);
+  border: 1px solid var(--mainText-accent-blue);
+  background-color: var(--rangeSlider-ocean);
+
+  display: flex;
   width: 100px;
   height: 30px;
-  background-color: #DC143C;
-  color: #ffffff;
-  border: 1px solid #ffffff;
+  margin: auto 30px;
+  letter-spacing: 1px;
+  border-radius: 4px;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
 }
 
 .PlayButton:hover {
-  border: 1px solid #DC143C;
-  background-color: #ffffff;
-  color: #DC143C;
+  color: var(--rangeSlider-ocean);
+  border: 1px solid var(--mainText-accent-blue);
+  background-color: var(--mainText-ordinary-white);
+
   cursor: pointer;
   transition: 0.3s ease all;
 }
 
+.PlayButton svg {
+  font-size: var(--fontSize-ButtonsIcons);
+  margin-right: 5px;
+}
+
 .PauseButton {
-  border-radius: 4px;
-  margin: auto 30px;
+  font-family: var(--font-ordinary);
+  font-size: var(--fontSize-text);
+  color: var(--mainText-ordinary-white);
+  border: 1px solid var(--mainText-accent-blue);
+  background-color: var(--rangeSlider-ocean);
+
+  display: flex;
   width: 100px;
   height: 30px;
-  border: 1px solid #DC143C;
-  background-color: #ffffff;
-  color: #DC143C;
+  margin: auto 30px;
+  letter-spacing: 1px;
+  border-radius: 4px;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
 }
 
 .PauseButton:hover {
-  background-color: #DC143C;
-  color: #ffffff;
-  border: 1px solid #ffffff;
+  color: var(--rangeSlider-ocean);
+  border: 1px solid var(--mainText-accent-blue);
+  background-color: var(--mainText-ordinary-white);
+
   cursor: pointer;
   transition: 0.3s ease all;
 }
 
+.PauseButton svg {
+  font-size: var(--fontSize-ButtonsIcons);
+  margin-right: 5px;
+}
+
 .StopButton {
-  border-radius: 4px;
-  margin: auto 30px;
+  font-family: var(--font-ordinary);
+  font-size: var(--fontSize-text);
+  color: var(--mainText-ordinary-white);
+  border: 1px solid var(--mainText-accent-blue);
+  background-color: var(--rangeSlider-ocean);
+
+  display: flex;
   width: 100px;
   height: 30px;
-  border: 1px solid #DC143C;
-  background-color: #ffffff;
-  color: #DC143C;
+  margin: auto 30px;
+  letter-spacing: 1px;
+  border-radius: 4px;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
 }
 
 .StopButton:hover {
-  background-color: #DC143C;
-  color: #ffffff;
-  border: 1px solid #ffffff;
+  color: var(--rangeSlider-ocean);
+  border: 1px solid var(--mainText-accent-blue);
+  background-color: var(--mainText-ordinary-white);
+
   cursor: pointer;
   transition: 0.3s ease all;
 }
+
+.StopButton svg {
+  font-size: var(--fontSize-ButtonsIcons);
+  margin-right: 5px;
+}
```

Comment:
I think it is better to get a class for <svg> and use here ".nameOfClass" instead of the name of the tag.

Reply:
done)

Reply:
Nothing changed